### PR TITLE
Enroll staff - !!!plz merger this after login_android branch!!!

### PR DESCRIPTION
**Pull request by Chan-Y**

### CODE REVIEW

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+public class EnrollStaffUseCase implements EnrollUserInputBoundary {
+
+    private EnrollUserOutputBoundary enrollUserOutputBoundary;
+    private final UserList userList;
+
+    public EnrollStaffUseCase(UserList userList) {
+        this.userList = userList;
+
+    }
+
```
Comment (evelyn323) [inline Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:16]: needs javadoc

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import constant.mangerSystem.UserType;
+import use_case.boundary.input.EnrollUserInputBoundary;
+import use_case.boundary.output.EnrollUserOutputBoundary;
+
+public class EnrollStaffUseCase implements EnrollUserInputBoundary {
+
+    private EnrollUserOutputBoundary enrollUserOutputBoundary;
+    private final UserList userList;
+
```
Comment (evelyn323) [inline Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:11]: needs javadoc

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
+package use_case.userList;
+
+import constant.mangerSystem.UserType;
+import use_case.boundary.input.EnrollUserInputBoundary;
+import use_case.boundary.output.EnrollUserOutputBoundary;
+
+public class EnrollStaffUseCase implements EnrollUserInputBoundary {
```
Comment (evelyn323) [inline Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:7]: class needs javadoc

File: Android/app/src/main/java/use_case/userList/UserList.java [hunk 4]
```diff
@@ -35,26 +37,27 @@ public UserList() {
...
      * Add user to this user list.
      *
      * @param user the user to add
      */
-    public static void addUser(User user) {
+    public void addUser(User user) {
         users.put(user.getId(), user);
     }
 
     public String addNewUser(String id, String name, String password) {
```
Comment (evelyn323) [inline Android/app/src/main/java/use_case/userList/UserList.java:53]: needs javadoc

File: Android/app/src/main/java/use_case/userList/UserList.java [hunk 5]
```diff
@@ -103,13 +107,35 @@ public Map getUsers(){
...
         }
         return builder.toString();
     }
 
-    public void SavetoFile(){this.readWriter.saveToFile(this.filepath,this.users);}
+    public void savetoFile() {
+        this.readWriter.saveToFile(this.filepath, users);
+    }
+
+    public void addStaff(String id, String name, String password, String userType, int salary) {
```
Comment (evelyn323) [inline Android/app/src/main/java/use_case/userList/UserList.java:123]: needs javadoc

**Review by evelyn323 — CHANGES_REQUESTED** [review]

**Review by ShaojieDong503 — COMMENTED** [review]
Review the LoginActivity

File: Android/app/src/main/java/use_case/userList/UserList.java (hunk 4 above)
Comment (Chan-Y) [reply Android/app/src/main/java/use_case/userList/UserList.java:53]: this method is redundant now, probably will be removed

File: Android/app/src/main/java/use_case/userList/UserList.java (hunk 5 above)
Comment (Chan-Y) [reply Android/app/src/main/java/use_case/userList/UserList.java:123]: working on

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java (hunk 3 above)
Comment (Chan-Y) [reply Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:7]: working on

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java (hunk 1 above)
Comment (Chan-Y) [reply Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:16]: working on

File: Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java (hunk 2 above)
Comment (Chan-Y) [reply Android/app/src/main/java/use_case/userList/EnrollStaffUseCase.java:11]: working on

**Pull request merged**